packages/sources/coinmetrics/src/transport/price-ws.ts

```diff
@@ -6,6 +6,9 @@ import { assetMetricsInputParameters, BaseEndpointTypes } from '../endpoint/pric
 
 const logger = makeLogger('CoinMetrics WS')
 
+// base currencies that break the ws connection
+export const invalid_base_currencies: string[] = []
```

Comment:
Can we use camel case, to be consistent with the rest of the file?

```suggestion
export const invalidBaseCurrencies: string[] = []
```

Reply:
Minor: I think it's more accurate to call these assets rather than currencies, 

```suggestion
export const invalidBaseAssets: string[] = []
```